test/schemas/form/Person.schema

```diff
@@ -0,0 +1,5 @@
+{
+  DOB: { domain: 'DateTime' },
+  FullName: { category: 'FullName' },
+  Mother: { category: 'Person' },
```

Comment:
Where is a `Father`? :smile:

Reply:
:trollface: `Mother` and `Father` is discrimination. According to the last directions, person may have two mothers :two_women_holding_hands: or tho fathers :two_men_holding_hands: